cmake/Modules/UseDamaris.cmake

```diff
@@ -0,0 +1,32 @@
+if(USE_DAMARIS_LIB)
+  if (Damaris_FOUND)
+    set(HAVE_DAMARIS 1)
+    message(STATUS "The Damaris library was found: ${Damaris_VERSION} ${Damaris_DIR}")
+    if (Damaris_HAS_HDF5)
+      message(STATUS "The Damaris library has HDF5 support: ${HDF5_VERSION}")
+    else()
+      message(STATUS "The Damaris library does NOT have HDF5 support")
+    endif()
+    if (Damaris_HAS_VISIT)
+      message(STATUS "The Damaris library has VisIt support: ${VisIt_VERSION}")
+    else()
+      message(STATUS "The Damaris library does NOT have VisIt support")
+    endif()
+    if (Damaris_HAS_CATALYST)
+      message(STATUS "The Damaris library has Catalyst support: ${Catalyst_VERSION}  ${Paraview_VERSION}")
+    else()
+      message(STATUS "The Damaris library does NOT have Catalyst support")
+    endif()
+  else()
+    message(STATUS "The Damaris library was requested but NOT found")
+  endif()
+else()  # User did not request Damaris support
+  if (Damaris_FOUND)
+    message(STATUS "The Damaris library was NOT requested but was found: ${Damaris_VERSION} ${Damaris_DIR}")
+  else()
+    message(STATUS "The Damaris library was NOT requested and was NOT found")
```

Comment:
Do we really need/want this particular diagnostic message?  Why does the user need to know anything about Damaris if they've not asked for it?

Reply:
in my opinion, no, but it was there in the PR I refactored.

Reply:
>> Do we really need/want this particular diagnostic message?
>
> in my opinion, no, but it was there in the PR I refactored.

Understood.  Maybe we could trim the diagnostics down a bit then?  I don't really care if we do that before or after merging this PR.